<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectWindow</class>
 <widget class="QWidget" name="ConnectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>407</width>
    <height>334</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ConnectWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="verticalWidget_3" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#verticalWidget_3{	
	
	background-image: url(:/MainWindow/Images/BackGroundCenter.png);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="styleSheet">
         <string notr="true">QTabWidget:pane{
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	border-radius: 10px;
}

QTabWidget:tab-bar{}

QTabBar::tab{
	     font: bold 11pt &quot;Marid Pro&quot;;
border: 1px solid black;
    background: white;
padding-top: 5px;
padding-bottom: 5px;
}

 QTabBar::tab:first{;
	border-top-left-radius: 12px;
	border-bottom-left-radius: 12px;		
min-width: 15ex;
 }

QTabBar::tab:last{
	border-top-right-radius: 12px;
	border-bottom-right-radius: 12px;		
min-width: 25ex;
 }

 QTabBar::tab:selected {
	color: white;
	background-color: black;	
 }</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideRight</enum>
        </property>
        <property name="usesScrollButtons">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <attribute name="title">
          <string>VPN</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="horizontalWidget" native="true">
            <property name="styleSheet">
             <string notr="true">
QLabel{
	color:  rgb(58, 78, 105);
	font:  11pt &quot;Marid Pro&quot;;
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>35</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>15</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLineEdit" name="cbServer">
                   <property name="minimumSize">
                    <size>
                     <width>230</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777212</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLineEdit {
	border-radius: 4px;
    border: 1px solid rgb(240, 240, 240, 255);
	font: bold 9pt &quot;Marid Pro&quot;;
	color: rgb(58, 78, 105);
 	background-color: rgb(255, 255, 255);
	padding-left: 20;
	image: url(:/MainWindow/Images/ComboBoxArrow.png);
	image-position: right;
	padding-right:5px;
}</string>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QGridLayout" name="gridLayout">
                   <item row="0" column="0">
                    <widget class="QLabel" name="lbIn">
                     <property name="maximumSize">
                      <size>
                       <width>30</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">font:  bold;</string>
                     </property>
                     <property name="text">
                      <string>In:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="lbOut">
                     <property name="styleSheet">
                      <string notr="true">font:  bold;</string>
                     </property>
                     <property name="text">
                      <string>Out:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLabel" name="lbDownloadSpeed">
                     <property name="minimumSize">
                      <size>
                       <width>110</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>0.00 B/s</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLabel" name="lbUploadSpeed">
                     <property name="text">
                      <string>0.00 B/s</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QLabel" name="lbDownloadBytes">
                     <property name="text">
                      <string>0.00 B</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QLabel" name="lbUploadBytes">
                     <property name="text">
                      <string>0.00 B</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btnConnect">
                 <property name="minimumSize">
                  <size>
                   <width>75</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>75</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
	border: 0px;
	
	background-image: url(:/MainWindow/Images/btnOff.png);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>20</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="lbIpAdress">
                   <property name="text">
                    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;IP&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;: 127.0.0.1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lbFlag">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lbStatusCon">
                   <property name="text">
                    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Status&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;: &lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#ff0000;&quot;&gt;Disconnected&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QCheckBox" name="cbEnableSmartDNS">
                 <property name="styleSheet">
                  <string notr="true">QCheckBox{
	color:  rgb(58, 78, 105);
	font: bold 9pt &quot;Marid Pro&quot;;
spacing: 15px;
}
QCheckBox::indicator:checked {
     image: url(:/MainWindow/Images/CheckBoxChecked.png);
 }

QCheckBox::indicator:unchecked {
     image: url(:/MainWindow/Images/CheckBox.png);
 }</string>
                 </property>
                 <property name="text">
                  <string>Enable Smart DNS</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <property name="styleSheet">
          <string notr="true">QLabel{
	color:  rgb(58, 78, 105);
	font:  11pt &quot;Marid Pro&quot;;
}</string>
         </property>
         <attribute name="title">
          <string>Smart DNS</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>60</number>
            </property>
            <property name="topMargin">
             <number>25</number>
            </property>
            <property name="rightMargin">
             <number>60</number>
            </property>
            <property name="bottomMargin">
             <number>25</number>
            </property>
            <item row="1" column="0" colspan="2">
             <widget class="QPushButton" name="pbSmartDnsHelp">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	border-radius: 6px;
	font:  12pt &quot;Marid Pro&quot;;
	color: rgb(255, 255, 255);
 	background-color: rgb(45, 182, 216, 255);
}</string>
              </property>
              <property name="text">
               <string>Help</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lbSmartDnsStatus">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Marid Pro'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Status&lt;/span&gt;: &lt;span style=&quot; color:#ff0000;&quot;&gt;Disable&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pbSmartDns">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
	border: 0px;
	
	background-image: url(:/MainWindow/Images/btnOff.png);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="footer" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#footer{
	background-image: url(:/MainWindow/Images/BackGroundBar.png);	
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	background-color: rgba(255, 255, 255, 0);
	image: url(:/MainWindow/Images/Logo.png);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSettings">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::PreventContextMenu</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
 background-color: rgba(54, 63, 72, 255);
font: bold 10pt &quot;Lucida Sans Unicode&quot;;
border: 0px;
}

QPushButton:hover {
}

QPushButton QMenu {
	color: white;
     background-color: black; /* sets background of the menu */
     border: 0px;
	border-radius: 4px;
 }

QPushButton QMenu::item {
	border: 1px solid transparent; /* reserve space for selection border */
 }
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="mainwindow.qrc">
          <normaloff>:/MainWindow/Images/Settings.png</normaloff>:/MainWindow/Images/Settings.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
